acceptance-tests/src/test/scala/ch/renku/acceptancetests/DatasetModificationFlow.scala

```diff
@@ -0,0 +1,48 @@
+package ch.renku.acceptancetests
+
+import ch.renku.acceptancetests.generators.Generators.Implicits._
+import ch.renku.acceptancetests.generators.Generators.paragraph
+import ch.renku.acceptancetests.model.datasets.{DatasetName, DatasetTitle}
+import ch.renku.acceptancetests.model.projects.ProjectDetails
+import ch.renku.acceptancetests.pages.ProjectPage
+import ch.renku.acceptancetests.tooling.{AcceptanceSpec, DocsScreenshots}
+import ch.renku.acceptancetests.workflows._
+
+import scala.language.postfixOps
+
+class DatasetModificationFlow extends AcceptanceSpec with Login with NewProject with RemoveProject with Datasets {
```

Comment:
Should we maybe have the `Spec` suffix in the name of the class? I think all the current specs have it and not sure if sbt will pick it up when it's not there...